### PR TITLE
Added playground for EmptyState

### PR DESCRIPTION
**Pull request by komal-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-Configure the library and set up the Playground Tab in the layout
-Configure the Playground tab

-Configure the Playground tab
-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2024-07-22 at 5 22 52 PM

Screenshot 2024-07-22 at 5 23 32 PM

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/EmptyState/playground/PlaygroundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                return { family: 'material', name: 'devices' };
+            case 'undefined':
+            default:
+                return undefined;
+        }
+    };
+    return (
+        <Stack alignItems={'center'} justifyContent={'center'} sx={{ width: '100%', height: '100%' }}>
+            <EmptyState
+                {...removeEmptyProps(rest)}
```
Comment (KomalGDhere-eaton) [inline docs/src/componentDocs/EmptyState/playground/PlaygroundPage.tsx:102]: Is removeEmptyProps required here ? We have not used removeEmptyProps in other playgroundPage component. Without this also playgroundPage working fine.

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good

**Review by KomalGDhere-eaton — APPROVED** [review]

File: docs/src/componentDocs/EmptyState/playground/PlaygroundPage.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        category: 'Optional Props',
+        defaultValue: 'inherit',
+        allowMuiColors: true,
+    },
+    // Other Configuration
+    {
+        id: 'showAction',
+        label: 'Show Actions',
+        type: 'boolean',
+        description: 'Whether to show action section or not',
```
Comment (ArshdeepSingh3-eaton) [inline docs/src/componentDocs/EmptyState/playground/PlaygroundPage.tsx:80]: change description to this `Additional components to render below the text`

Comment (komal-eaton) [reply docs/src/componentDocs/EmptyState/playground/PlaygroundPage.tsx]: Changed

**Pull request merged**